src/genie/libs/parser/iosxr/tests/ShowImDampening/cli/equal/golden_output1_expected.py

```diff
@@ -0,0 +1,41 @@
+expected_output = {
+                    'interface': {
+                        'GigabitEthernet0/0/0/0': {
+                            'protocol': {
+                                'not_present': {
```

Comment:
Makes sense but I just wondered if that was a hardcoded key of "1:" ? It looks a bit like a list so I wondered if there was special way of coding of it or just treat it as normal set of dict keys. I think I am right in thinking we couldn't have more than one index (as we have the interface as a unique key) so hard coding it exactly like that should be okay?

Reply:
the number `1` will be incremented if we see same interface as multiple. the index number will start with `1` and then `2`, `3` and so on.

We usually don't have a list in the middle of dictionary. (only leaf has list as value). So, that's why we are doing `index` thing for this case.

Reply:
Since Daniel's changes, the empty key values now report None instead. I assume this would negate the need for using a index here? See conversation with Daniel higher up where those changes are proposed. If we need to convert from None to index then please let me know. Thanks.

Reply:
Hi @gwoodwa1, we still need the index instead of using protocol otherwise if there are two values with `None`, one would overwrite the other

Reply:
confirmed the change